cmd/tk/workflow.go

```diff
@@ -56,10 +64,7 @@ func applyCmd() *cobra.Command {
 			log.Println("Warning: There are no differences. Your apply may not do anything at all.")
 		}
 
```

Comment:
This won‘t work because flags cannot be registered in Run, they need to exist before.

Instead, just call the function outside of Run and use the variable in here

Reply:
Same for Delete